components/Card.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export default function card({ voc }) {
+  return (
+    <>
+      <StyledSection>
+        <p>{voc.word}</p>
+      </StyledSection>
+    </>
```

Comment:
You don't need to wrap your component's return content with an empty fragment. Since there's only one element returned, you can directly return the `StyledSection`.

Reply:
You're sooo right 👍